holmes/plugins/toolsets/bash/bash_toolset.py

```diff
@@ -22,120 +18,18 @@
     ToolsetTag,
 )
 from holmes.plugins.toolsets.bash.common.bash import execute_bash_command
-from holmes.plugins.toolsets.bash.common.config import BashExecutorConfig
-from holmes.plugins.toolsets.bash.kubectl.constants import SAFE_NAMESPACE_PATTERN
-from holmes.plugins.toolsets.bash.kubectl.kubectl_run import validate_image_and_commands
 from holmes.plugins.toolsets.bash.parse_command import make_command_safe
-from holmes.plugins.toolsets.utils import get_param_or_raise
 
 
 class BaseBashExecutorToolset(Toolset):
```

Comment:
I see that there are 4 evals which failed and marked as easy. Looks like it is due to the bash toolset change. How we gonna fix it?